staff.js

```diff
@@ -1,9 +1,11 @@
 
 
 class Staff{
-	constructor(name){
+	constructor(currentHotel , currentEmpId){
```

Comment:
we are not adding an employee id at this time from the outside code.  Please remove this.